probeinfo_api.yaml

```diff
@@ -734,6 +750,125 @@ components:
+          value:
+            type: [ string, null ]
+            required: true
+
+    ExpirationPolicy:
+      type: object
+      description: |
+        Currently not in use.  Various options controlling data lifecycle.
+      additionalProperties: false
+      properties:
+        collect_through_date:
+          type: string
+          description: |
+            Currently not in use.  If present, the pipeline will reject new data with submission_timestamp after the
+            given date, sending it to error output.
+          pattern: "^[0-9]{4}-[0-9]{2}-[0-9]{2}$"
+        delete_after_days:
+          type: integer
+          description: |
+            Currently not in use. The number of days to retain decoded ping data received for this application.
+            If not specified, retention will be unlimited.
+
+    JweMappings:
+      type: array
+      description: |
+        Currently not in use.  Mappings of encrypted JWE field paths to destinations where the value decrypted by the
+        pipeline should be placed; initial use case is Account Ecosystem Telemetry; paths must be in
+        [JSON Pointer format](https://tools.ietf.org/html/rfc6901) like '/payload/ecosystemAnonId'.
+      items:
+        type: object
+        additionalProperties: false
+        properties:
+          decrypted_field_path:
+            type: string
+            required: true
+            default: ""
+          source_field_path:
+            type: string
+            required: true
+            pattern: "^/.*$"
+            default: "/payload"
+
+    MozPipelineMetadata:
+      type: object
+      description: |
+        Currently not in use.
+        Container for per-doctype metadata that can affect how pings are processed in the pipeline.
+      additionalProperties: false
+      properties:
+        expiration_policy:
+          $ref: "#/components/schemas/ExpirationPolicy"
+        jwe_mappings:
+          $ref: "#/components/schemas/JweMappings"
+        override_attributes:
+          $ref: "#/components/schemas/OverrideAttributes"
+        submission_timestamp_granularity:
+          type: string
+          description: |
+            Currently not in use.
+            If specified, the submission_timestamp field will be truncated to the specified
+            granularity in the pipeline before being output to BigQuery; this can be used to reduce the potential for
+            using time-based attacks to correlate datasets using different client-level identifiers; see Java's
+            ChronoUnit for additional granularities that could be considered for inclusion; implemented for bug 1742172.
+          enum:
+            - millis
+            - seconds
+            - minutes
+            - hours
+            - days
+
+    MozPipelineMetadataOverride:
+      type: object
+      description: |
+        Currently not in use.
+        Allows ping level override of metadata values.
+      patternProperties:
```

Comment:
This isn't showing up in the rendered docs. Can this be expressed with `additionalProperties` instead like https://github.com/mozilla/probe-scraper/blob/58667b28fd0e13e49e89afb527af2483f03ab598/probeinfo_api.yaml#L253-L257 ?

Reply:
And can we add a comment here to indicate that we're expecting the keys here to be ping names?

Reply:
Yes, that looks great. Is this working with the current code? I rendered these locally but may have been using an outdated version of `redoc-cli`.

Reply:
We should also specify here that ping name here is specifically referring to the document type (which is `kebab-case`) rather than the BigQuery table name (which is `snake_case`).

Reply:
Update has been pushed.